program/engagement/transition.md

```diff
@@ -54,7 +54,7 @@ The DTA may co-fund transition activities with agencies.
 
 ## Enabling Services 
 
-* The community.desktop.gov.au community portal: for asking questions, FAQ, and understanding which agencies are using the [Blueprint](/blueprint).  
+* The [community portal](community.desktop.gov.au): for asking questions, FAQ, and understanding which agencies are using the [Blueprint](/blueprint).  
```

Comment:
Hyperlinks requires the protocol when you're linking to a different domain, should be:

`[community portal](https://community.desktop.gov.au/)`

Reply:
fixed :-)